3.0.0-alpha.1: missing dependency / types for `okie`

### Describe the bug

Cannot find module 'okie' or its corresponding type declarations.

The `okie` package appears to have types, but is not being bundled for some reason. Vite's `dist/node/index.d.ts` has:
```
import type { Worker as Worker_2 } from 'okie';
```

### Reproduction

https://github.com/sveltejs/kit/tree/vite-3

### System Info

```shell
Vite 3.0.0-alpha.1
```


### Used Package Manager

pnpm

### Logs

```shell
Error: node_modules/.pnpm/vite@3.0.0-alpha.1/node_modules/vite/dist/node/index.d.ts(50,41): error TS2307: Cannot find module 'okie' or its corresponding type declarations.
Error: node_modules/.pnpm/vite@3.0.0-alpha.1/node_modules/vite/types/importGlob.d.ts(1,29): error TS2307: Cannot find module 'okie' or its corresponding type declarations.
```


### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitejs/vite/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitejs/vite/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitejs.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/vitejs/vite/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Vite issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitejs/vite/discussions) or join our [Discord Chat Server](https://chat.vitejs.dev/).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

--- a/packages/vite/types/importGlob.d.ts
+++ b/packages/vite/types/importGlob.d.ts
@@ -1,5 +1,3 @@
-import type { Worker } from 'okie'
-
 export interface ImportGlobOptions<
   Eager extends boolean,
   AsType extends string
@@ -35,7 +33,7 @@ export type GeneralImportGlobOptions = ImportGlobOptions<boolean, string>
 export interface KnownAsTypeMap {
   raw: string
   url: string
-  worker: Worker<any, any>
+  worker: Worker
 }
 
 export interface ImportGlobFunction {

